apps/server/src/modules/tool/service/validation/external-tool-validation.service.spec.ts

```diff
@@ -129,6 +132,32 @@ describe('ExternalToolValidation', () => {
 				});
 			});
 		});
+
+		describe('when external tool config is lti11Config', () => {
+			describe('when there is no secret', () => {
+				const setup = () => {
+					const externalLti11ToolDOWithoutSecret: ExternalToolDO = externalToolDOFactory
+						.withLti11Config({ key: 'lti11Key', secret: undefined })
+						.buildWithId();
+					return { externalLti11ToolDOWithoutSecret };
+				};
+				it('should throw validation error', async () => {
```

Comment:
please us empty lines ... then it is more readable

Reply:
Do you want to solve this comment? @IgorCapCoder

Reply:
done :)